src/llmad/utils.py

```diff
@@ -0,0 +1,19 @@
+import warnings
+
+
+def identify_file_type(file_path: str) -> str:
```

Comment:
we can use magic library for this

Reply:
Indeed, I have now implemented `indentify_mime_type` based on magic